### PR TITLE
Sentence added under Playground

### PR DESCRIPTION
**Pull request by Suman373**
Added a small sentence which would be more appropriate than the previous pr with a big definition

## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

### Why is this valuable (or not)?
 Valuable
## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)
- [ ] Search for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Suman373** [comment]
If this gets approval, I would like to close the previous pr which was labelled for review

File: README.md [hunk 1]
```diff
@@ -182,6 +182,8 @@ Free Podcasts and Screencasts:
 
 ### Programming Playgrounds
 
+Write, compile and run your code. Try it out !
```
Comment (CleanMachine1) [inline README.md:185]:
```suggestion
Write, compile and run your code. Try it out!
```

Comment (CleanMachine1) [reply README.md:185]: Not a suggestion of the text, but a "syntax error"

Comment (Suman373) [reply README.md:185]: Oh, mb. Ty

**Review by davorpa — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -182,6 +182,8 @@ Free Podcasts and Screencasts:
 
 ### Programming Playgrounds
 
+Write, compile and run your code. Try it out!
```
Comment (CleanMachine1) [inline README.md:185]:
```suggestion
Write, compile and run your code within a browser. Try it out!
```
@davorpa @Suman373 What do we think of this.

Comment (Suman373) [reply README.md:185]: Looks better!!

**Review by CleanMachine1 — APPROVED** [review]

**Review by LorenzoQuiroz — APPROVED** [review]
LGTM; appropriate and helpful sentence

**Pull request merged**